interface.go

```diff
@@ -26,6 +26,7 @@ type Manager interface {
 	List(context.Context, Filter) ([]Process, error)
 	Group(context.Context, string) ([]Process, error)
 	Get(context.Context, string) (Process, error)
+	Clear(context.Context) error
```

Comment:
are there actually cases where this can return an error?

Reply:
Not many, however, this exists because sometimes it can, but mostly in the REST implementation, since we have code there to handle errors with requests and such (https://github.com/mongodb/jasper/pull/20/files#diff-b215740dd688afb2721c9daf2c1287e3R216).